plugins/arDominionB5Plugin/modules/accession/templates/_events.php

```diff
@@ -6,10 +6,14 @@
   <table class="table table-bordered mb-0 multi-row">
     <thead class="table-light">
       <tr>
-        <th id="accession-events-type-head" style="width: 20%">
+	<style <?php echo __(sfConfig::get('csp_nonce', '')); ?>>
```

Comment:
This style can be replaced with the BS5 w-35 style. Leave the id's in place as they existed beforehand.